KillProofModule.cs

```diff
@@ -182,7 +239,21 @@ protected override async Task LoadAsync()
             }
             return false;
         }
-
+        private void SendToChat(string message) {
+            System.Windows.Forms.Clipboard.SetText(message);
```

Comment:
Oh, I would recommend you use the `ClipboardUtil`.  You can see an example of how it is used here:

https://github.com/blish-hud/Blish-HUD/blob/2a3b9c101d37d3571bb2f78f7f7e34303b2848a7/Blish%20HUD/Controls/TextInputBase.cs#L569-L574

We found out the hard way that setting the clipboard text is actually not always reliable and will sometimes throw an exception.  This will help prevent that from happening and once it completes you can confirm it went through successfully and then run that task below.

Reply:
You'll just need to just include the Nuget package that declares the reference for the library we use in the core application.

`Install-Package AsyncClipboardService -Version 1.7.1`